server/genserver.go

```diff
@@ -95,22 +98,20 @@ func (s *Server) Serve() error {
 		s.l.Info("please provide a kvconfig.yml file to enable app log routing")
 	}
 
-	if tracingToken := os.Getenv("TRACING_ACCESS_TOKEN"); tracingToken != "" {
-		ingestUrl := os.Getenv("TRACING_INGEST_URL")
-		if ingestUrl == "" {
-			ingestUrl = "https://ingest.signalfx.com/v1/trace"
+	tracingToken := os.Getenv("TRACING_ACCESS_TOKEN")
+	isLocal := os.Getenv("_IS_LOCAL") == "true"
+	if tracingToken != "" || isLocal {
+		ingestURL := os.Getenv("TRACING_INGEST_URL")
+		if ingestURL == "" {
+			ingestURL = "https://ingest.signalfx.com/v1/trace"
```

Comment:
This URL might not be supported any more. If it's not, we should just remove it and error if there's no ingestURL

Reply:
this felt like code smell anyways, happy to drop it

Reply:
https://github.com/Clever/wag/pull/245/commits/17ef4136d103054998e175feff6ec011777525c0